website/cue/reference/remap/functions/is_nullish.cue

```diff
@@ -3,10 +3,17 @@ package metadata
 remap: functions: is_nullish: {
 	category: "Type"
 	description: """
-		Determines whether `value` is nullish, where nullish denotes the absence of a
-		meaningful value.
+		Determines whether `value` is nullish. Returns `true` if the specified `value` is `null`,
+		an empty string, a string containing only whitespace, or the string `"-"`. Returns `false` otherwise.
 		"""
 
+	notices: [
+		"""
+			This function behaves inconsistently: it returns `false` for empty arrays (`[]`) and objects (`{}`),
+			but `true` for empty strings (`""`) and `null`. Consider using `is_empty` as a better alternative.
```

Comment:
I think `is_empty` and `is_nullish` have two different use-cases (`is_nullish` is meant to be used with parsing functions to test if a given field is "null") so I'm not about the last statement that `is_empty` is a better alternative 🤔 

Otherwise 👍 on clarifying exactly what values are considered nullish.